themes/custom/droopler_theme/scss/components/_megamenu_overrides.scss

```diff
@@ -0,0 +1,13 @@
+.alert {
+  .close {
+    padding: 0.75rem 1.25rem !important;
```

Comment:
jedyne do czego @grzegorz-pietrzak-droptica może mieć życzenie, to wrzucenie tego w zmienne (tak jak inne scss, to z !default) :thinking:

Reply:
ten kod jak mniemam to tylko taki tymczasowy fix. To jest defaultowy padding z bootstrapa i jak usuniemy problem z megamenu to będzie można ten fragment wywalić.

Reply:
niech zostanie już, będziemy się tym martwić faktycznie po alpha2